src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -130,6 +131,8 @@
     --pf-c-menu-toggle--active--BackgroundColor: var(--pf-c-menu-toggle--m-primary--active--BackgroundColor);
     --pf-c-menu-toggle--m-expanded--Color: var(--pf-c-menu-toggle--m-primary--m-expanded--Color);
     --pf-c-menu-toggle--m-expanded--BackgroundColor: var(--pf-c-menu-toggle--m-primary--m-expanded--BackgroundColor);
+
+    border-radius: var(--pf-c-menu-toggle--m-primary--BorderRadius);
```

Comment:
wdyt about adding this to the base toggle and using a var for the primary variation? then we can add that for other variations if we add them.

Reply:
oh sorry what I meant was like:

```
.pf-c-menu-toggle {
  --pf-c-menu-toggle--BorderRadius: 0;
  --pf-c-menu-toggle--m-primary--BorderRadius: var(--pf-global--BorderRadius--sm);

  border-radius: var(--pf-c-menu-toggle--BorderRadius);

  &.pf-m-primary {
    --pf-c-menu-toggle--BorderRadius: var(--pf-c-menu-toggle--m-primary--BorderRadius);
  }
}
```

Then we can just update the `--BorderRadius` var any time we need to introduce a border-radius to a variation instead of defining the property per variation.